examples/core/gated_gcn/main.py

```diff
@@ -0,0 +1,124 @@
+import argparse
```

Comment:
I don't quiet understand your meaning, do you mean I should a test If the test takes less than 5 mins to finish?

Reply:
I mean how long does the training process take? More than 5 mins or not.

But never mind, I think it is better to add this to regression test instead of unit test. No action required for you.